src/styles/themes.scss

```diff
@@ -91,6 +92,7 @@
 
   --theme-mode-indicator: #{vars.$dark-mode-indicator};
   --theme-mode-icon: #{vars.$white};
+  --size-selector-border: #{vars.$dark-primary-4};
```

Comment:
I think for consistency, since all the other variables start with `--theme-`, this one should too